server/schedule/operator.go

```diff
@@ -647,6 +726,76 @@ func getIntersectionStores(a []*metapb.Peer, b []*metapb.Peer) map[uint64]struct
 	return intersection
 }
 
+// CreateScatterRegionOperator creates an operator that scatters the specified region.
+func CreateScatterRegionOperator(desc string, cluster Cluster, origin *core.RegionInfo, replacedPeers, targetPeers []*metapb.Peer) *Operator {
```

Comment:
This operator is not very general, maybe it's more readable in the old place?

Reply:
To keep consistent with others, I think it's ok.